libraries/Microsoft.Bot.Schema/Teams/MeetingNotificationResponse.cs

```diff
@@ -7,26 +7,26 @@
 namespace Microsoft.Bot.Schema.Teams
 {
     /// <summary>
-    /// Container for <see cref="TeamsMeetingNotificationRecipientFailureInfo"/>, which is the result of a
-    /// failure to notify recipients of a <see cref="TeamsMeetingNotification"/>.
+    /// Specifies Bot meeting notification response.
+    /// Contains list of <see cref="NotificationRecipientFailureInfo"/>.
     /// </summary>
-    public partial class TeamsMeetingNotificationRecipientFailureInfos
+    public partial class MeetingNotificationResponse
```

Comment:
no reason for this to be a partial class from my perspective

Reply:
Will update it in next iteration.